style.css

```diff
@@ -1,30 +1,58 @@
 
 * {box-sizing: border-box;}
 
-.footer, .header, .main {
+.header, .main, .footer {
   display: block;
 }
 
-.container, .wrap {
+.header, .main__wrap, .footer__wrap {
+  margin: 0 auto;
   position: relative;
+  z-index: 1;
+}
+
+.header {
+  height: 100px;
 }
 
-.container {
+.header_left {
+  left: 0;
+  float: left;
+}
+
+.header_right {
+  right: 0;
+  float: right;
+}
+
+.header_left, .header_right {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;                  
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;            
   height: inherit;
-  margin: 0 auto;
-  z-index: 1;
+  width: 50%;
 }
 
-@media (min-width: 375px) { 
-  .container {
+.header_right {
+  	justify-content: flex-end;
+}
+
+
+@media (min-width: 320px) { 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Mandri_Firsttask/pullRequest?prid=5478414)